README.md

```diff
@@ -7,17 +7,95 @@ copyright: Copyright (c) 2010-2020 K Team. All Rights Reserved.
 
 # Introduction
 
-This is a readme file for _K developers_.
+The K Framework is a tool for designing and modeling programming languages
+and software/hardware systems.
+At the core of the K Framework is a programming, modeling, and specification
+language called K.
+The K Framework includes tools for compiling K specifications to build
+interpreters, model checkers, verifiers, associated documentation, and more.
+
+## Preface
+
+This is a readme file for _K developers_. Users should feel comfortable using
+the command line, as we do not provide GUI tools at this time.
 
 _K-based tool users_ should:
 
 1.  Consult their tool documentation for build/installation instructions.
 2.  If needed, download a [packaged release](https://github.com/kframework/k/releases/)
     of the K Framework as part of their tool setup process.
 
+If you are interested in quickly trying out the K Framework without building
+from source, please see our
+[quick start guide](k-distribution/INSTALL.md#quick-start-guide-for-ubuntu-bionicfocal).
+
 The rest of this file assumes you intend to build and install the K Framework
 from source.
 
+## Contents
+
+1.  [Supported System Configurations](#supported-system-configurations)
+2.  [Prerequisite Install Guide](#prerequisite-install-guide)
+3.  [Build and Install Guide](#build-and-install-guide)
+4.  [IDE Setup](#ide-setup)
+5.  [Running the Test Suite](#running-the-test-suite)
+6.  [Changing the KORE Data Structures](#changing-the-kore-data-structures)
+7.  [Building the Final Release Directory/Archives](#building-the-final-release-directoryarchives)
+8.  [Compiling Definitions and Running Programs](#compiling-definitions-and-running-programs)
+9.  [Troubleshooting](#troubleshooting)
+
+# Supported System Configurations
+
+We release packages for the following system configurations given in order
+of kernel, CPU architecture, and operating system/package manager.
+These configurations are known to be able to build and run the K Framework:
+
+*   Linux (x86-64) Ubuntu Bionic Beaver
+*   Linux (x86-64) Ubuntu Focal Fossa
+*   Linux (x86-64) Arch Linux
+*   Linux (x86-64) Debian Buster
+*   Linux (x86-64) Platform Indepdent Binary (tested on Ubuntu)
+*   macOS (x86-64) Brew
+
+NOTE: users of the platform independent binary must install any needed
+runtime dependencies separately.
+
+The following system configurations have **no direct support** (but see notes
+below about virtual machine/emulation support):
+
+*   Windows (any architecture)
+*   macOS (M1)
+
```

Comment:
This is packaging information and should remain in k-distribution/INSTALL.md. We do not want to duplicate information whenever possible, rather link to the one true source.